pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -120,14 +126,14 @@ def upload_to_s3(self, file, table, tmp_dir=None):
 
     def create_schema(self, schema):
         sql = 'CREATE SCHEMA IF NOT EXISTS {}'.format(schema)
-        self.query(sql)
+        self.query(sql, query_tag_props={'schema': schema, 'table': None})
```

Comment:
Given how the `table` value is being dealt with in the `open_connection` method, there is no need to mention the table here `'table': None`.

Reply:
fixed by 26febe82084667ac7e7db47299147e0dfa284257